modules/next/tests/modules/next_tests/next_tests.info.yml

```diff
@@ -1,7 +1,7 @@
 name: Next.js tests
 description: Configures testing for Next.js
 type: module
+core_version_requirement: ^10 || ^11
```

Comment:
@apathak18 I think we can remove `core_version_requirement` key all-together. As this is not required for tests module. Re: https://www.drupal.org/node/3096609

Reply:
I guess we need to keep core_version_requirement  here https://github.com/apathak18/next-drupal/actions/runs/10260927615/job/28387772151#step:12:182